export function handleError(errorMessage) {
  alert(errorMessage);
}
